tests/test_modeling_common.py

```diff
@@ -695,3 +705,96 @@ def test_model_from_pretrained(self):
             self.assertEqual(model.config.output_attentions, True)
             self.assertEqual(model.config.output_hidden_states, True)
             self.assertEqual(model.config, config)
+
+
+@require_torch
+class UtilsFunctionsTest(unittest.TestCase):
+
+    # tests whether the top_k_top_p function behaves as expected
+    def test_top_k_top_p_filtering(self):
+        if is_torch_available():
+            sys.path.insert(1, "../src")
```

Comment:
seems that the path '../src' has to be inserted to the PYTHONPATH before import since 
`from transformers import top_k_top_p_filtering` does not work.